libvirt/README.md

```diff
@@ -19,13 +19,25 @@ You must have a Linux/KVM system with libvirt installed and the following tools:
 Assume that you have a 'default' network and storage pools created in libvirtd system instance (`qemu:///system`). However,
 if you have a different pool name then the scripts should be able to handle it properly.
 
+The script [config_libvirt.sh](config_libvirt.sh) is tested on Ubuntu 20.04 amd64/s390x VSI, please config your Ubuntu with it, run the script from the root of this repo:
+```bash
+./libvirt/config_libvirt.sh
+```
+
+**Note** when your host VM is Ubuntu 22.04 with s390x arch, you will fail to create cluster with error:
+```
+...
+libvirt.libvirtError: unsupported configuration: ps2 is not supported by this QEMU binary
+```
+TODO: Need a fix from kcli and check this issue again.
+
```

Comment:
Now Karim has merged your fix I tested this on ubuntu 22.04 and it seems to work creating the cluster, so I think we can remove this bit. Thanks to you both!